cmd/frontend/graphqlbackend/external_accounts_test.go

```diff
@@ -104,6 +107,22 @@ func TestExternalAccounts_AddExternalAccount(t *testing.T) {
 			wantErr:        false,
 			accountDetails: `{"username": "alice", "password": "test"}`,
 		},
+		"SOAP account can only be added to site admins": {
```

Comment:
Whenever I see SOAP, [I imagine this](https://www.w3.org/TR/soap/). Can we instead use full naming to avoid ambiguity in IAM space?
```suggestion
		"Sourcegraph operator account can only be added by site admins": {
```